### DISCUSSION TITLE
Connecting 2 drives , 3.5" and 5.25" multiplexer

### DISCUSSION THREAD

**Discussion by gilmod714** [keirf/greaseweazle#512] [Ideas]
I made this small PCB for connecting 2 switchable drives to the GW .. Maybe someone will find it useful .. I housed it inside a damaged commodore 1541-case I had ..

![schem--FINLA-wrx](https://github.com/user-attachments/assets/10fc1a6a-4ddb-4ded-97bd-4713db8e9290)

![IMG_20241206_022858 (Medium)](https://github.com/user-attachments/assets/d0df4755-4b5d-4f30-b679-b7f8059a2963)

![IMG_20241206_022912 (Medium)](https://github.com/user-attachments/assets/3829e2ed-7906-4bdd-8b56-315c2a470d83)

Finished idea .. works great!

![IMG_20241209_032124 (Medium)](https://github.com/user-attachments/assets/5d355651-36ce-490d-84fb-8d2c355c2e95)

**Comment by Magneticdud** [comment]
hi, i don't have any hardware in my hands to test, and i never used this software and i am a complete noob but is a mux actually needed? (i want to build something like you, a single box with 3.5 and 5.25 drives)

Because the [docs](https://github.com/keirf/greaseweazle/wiki/V4.1-Setup#drive-terminations) say "It is also important that only one drive in a bussed setup has 150-ohm terminations: Greaseweazle V4.1 cannot drive into multiple strong pull-ups in parallel." and "if using a 5.25- or 8-inch drive in a multi-drive setup, you probably do not need the termination resistors" which let me think that it could access multiple drives at a time.

then the software has [options to specify which drive in the bus](https://github.com/keirf/greaseweazle/wiki/Drive-Select)

**Reply by gilmod714** [reply]
hello @Magneticdud

I have designed and built this 2 drive switch and it works great... just follow the schematic up there .. no need to overthink stuff :)

cheers
